src/utils/gravity.ts

```diff
@@ -2,13 +2,25 @@ import fetch from "node-fetch"
 import { Config } from "../config"
 
 class Gravity {
-  static HOSTS = {
-    staging: "stagingapi.artsy.net",
-    production: "api.artsy.net",
+  static BASE_URL = `https://api.artsy.net/`
```

Comment:
I _think_ the separate staging/production URLs were probably there with the intention of being able to get a token from either environment. I think it's useful to get either to plug into Insomnia/postman to query against metaphysics. I'm not actually sure the code before these changes would _actually_ get you a token from either environment...but I think that's probably why we kept track of both of them originally.

As far as I can tell, with these changes we can only get tokens from production. Do you have any thoughts about being able to toggle this between staging/prod easily?

Reply:
Yeah, that's a good question. I think it's valuable to point to either environment but am just not clear on the best API.

- Should we have env-aware keys for storing tokens?
- For any command backed by the Artsy API, what does the flag look like? Is it `--staging` or `artsy-host=https://apistaging.artsy.net`, something else?

Reply:
* +1 on env-aware keys. It seems like it'd be easy to lose track of where your token came from otherwise.
* I prefer `--staging` or an "environment" flag like `--env staging` to spelling out the actual host, for less typing/memorization of the URL. 

Also, for clarity - I don't think supporting both environments is a requirement for merging this PR.